### ISSUE TITLE
OHLCV Order Different for Different Exchanges

### ISSUE THREAD

**Issue by IMikhelson** [ccxt/ccxt#3551]
- OS: Windows 10
- Programming Language version: Python 3.6
- CCXT version: 1.17.65
- Exchange: Coinbase Pro, Binance, Kraken (maybe others, haven't tried yet)
- Method: fetch_ohlcv

I am trying to write some code using technical analysis, so I query OHLCV data.  However, the data is returned in either ascending or descending (by time) order, which makes the analysis more difficult, since I have to have an if statement by exchange to flip the order for some and not others.  Specifically, the ones I have tried are Coinbase Pro, Binance, and Kraken.  Binance and Kraken return the data in temporally ascending order, while Coinbase Pro returns in descending order.  Since I figured the whole point of CCXT was to allow people to interact uniformly across exchanges, I think this is worth looking into.

Thank you!

**Comment by IMikhelson** [comment]
After running a quick test, I found the following results.  This does not include exchanges that were unavailable at the moment, that require an API key to query such data, or that threw an error upon this request.  I used a '1h' resolution, as most exchanges support it.

acx : ascending order
anybits : ascending order
bigone : ascending order
binance : ascending order
bit2c : ascending order
bitbank : ascending order
bitbay : ascending order
bitfinex : ascending order
bitfinex2 : ascending order
bitkk : ascending order
bitmex : ascending order
bitsane : ascending order
bitstamp : ascending order
bittrex : ascending order
bitz : ascending order
bl3p : ascending order
braziliex : ascending order
btcalpha : descending order
btcmarkets : ascending order
btctradeim : ascending order
btcturk : descending order
ccex : ascending order
chilebit : ascending order
cobinhood : ascending order
coinbasepro : descending order
coinex : ascending order
coinfalcon : ascending order
coinfloor : ascending order
coinnest : ascending order
coinone : ascending order
crypton : ascending order
cryptopia : ascending order
deribit : ascending order
dsx : ascending order
ethfinex : ascending order
exx : ascending order
fcoin : descending order
foxbit : ascending order
fybse : ascending order
fybsg : ascending order
gatecoin : ascending order
gdax : descending order
hadax : descending order
hitbtc : ascending order
hitbtc2 : ascending order
huobipro : descending order
ice3x : ascending order
independentreserve : ascending order
indodax : ascending order
itbit : ascending order
kraken : ascending order
kucoin : ascending order
kuna : ascending order
mercado : ascending order
negociecoins : ascending order
nova : ascending order
okcoinusd : ascending order
okex : ascending order
paymium : ascending order
qryptos : ascending order
quadrigacx : ascending order
southxchange : ascending order
surbitcoin : ascending order
tidebit : ascending order
tidex : ascending order
urdubit : ascending order
vaultoro : ascending order
vbtc : ascending order
wex : ascending order
zb : ascending order
